src/javascript/eslint.ts

```diff
@@ -97,6 +97,12 @@ export interface EslintOverride {
    */
   readonly files: string[];
 
+  /**
+   * Pattern(s) to exclude from this override.
+   * If a file matches any of the excluded patterns, the configuration won’t apply.
+   */
+  readonly excludedFiles?: string | string[];
```

Comment:
In jsii languages this doesn't work very well. Let's just use an array here.

```suggestion
  readonly excludedFiles?: string[];
```